devtools/x/src/generate_summaries.rs

```diff
@@ -11,13 +11,24 @@ use std::{
     fs,
     path::{Path, PathBuf},
 };
-use structopt::StructOpt;
+use structopt::{clap::arg_enum, StructOpt};
+
+arg_enum! {
+    #[derive(Debug, Copy, Clone)]
+    pub enum OutputFormat {
+        Toml,
+        Json,
+    }
+}
 
 #[derive(Debug, StructOpt)]
 pub struct Args {
     #[structopt(name = "OUT_DIR")]
     /// Directory to output summaries to (default: target/summaries)
     out_dir: Option<PathBuf>,
+    #[structopt(name = "OUTPUT_FORMAT")]
+    /// Output in text, toml or json
+    output_format: Option<OutputFormat>,
```

Comment:
I think the only supported output formats are toml and json? If so I think you can just use a non-Option enum here and set a default: https://github.com/diem/diem/blob/39e4d7a064ea4b9d071c56d9df5099b655c4b8f8/devtools/x/src/cargo/build_args.rs#L85-L87

Reply:
simplified : o tested, it works!